spec/services/usps_in_person_proofing/eipp_helper_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe UspsInPersonProofing::EippHelper do
+  subject(:subject) { described_class }
+
+  describe '#extract_vector_of_trust' do
+    let(:vector_of_trust) { ['C1.P1.Pe'] }
+    let(:sp_session) { { 'vtr' => vector_of_trust, 'acr_values' => nil } }
+    it 'returns the vector_of_trust' do
+      expect(subject.extract_vector_of_trust(sp_session)).to eq(vector_of_trust)
+    end
+
+    context 'when the user enters the flow without a service provider' do
```

Comment:
IIRC, this shouldn't be possible in production.  It could occur if someone on the team is testing IPP in a non-production environment.